app/views/guides_for_parents_carers_students/index.html.erb

```diff
@@ -34,7 +34,7 @@
 
     <ul class="govuk-list govuk-list--spaced">
       <li>
-        <%= govuk_link_to t('devices_guidance.4g_user_guidance.title'), '/devices/4g-user-guidance'%>
+        <%= govuk_link_to 'How to get started with your 4G wireless router', '/devices/managing-your-4g-wireless-routers'%>
```

Comment:
Removed from en.yml, since it's copy-and-pasted elsewhere for this link text and this section is being removed.